src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/migration/Cas1InformationReceivedMigrationJob.kt

```diff
@@ -35,6 +36,9 @@ class Cas1InformationReceivedMigrationJob(
             clarificationNoteRepository.updateHasDomainEvent(clarificationNote.id)
           } catch (e: IllegalStateException) {
             migrationLogger.error(e.message ?: "An unknown error occurred")
+          } catch (e: RuntimeException) {
+            val errorMessage = e.message ?: "An unknown error occurred"
+            migrationLogger.error("Can't create domain event for ${clarificationNote.id} - $errorMessage")
```

Comment:
can you pass the exception into the logger please a second arg, then we'll get the stack trace